internal/url/url_test.go

```diff
@@ -14,10 +14,20 @@ var _ = Suite(&URLSuite{})
 
 func (s *URLSuite) TestMatchesScpLike(c *C) {
 	examples := []string{
+		// Most-extended case
 		"git@github.com:james/bond",
-		"git@github.com:007/bond",
+		// Most-extended case with port
 		"git@github.com:22:james/bond",
+		// Most-extended case with numeric "username"
```

Comment:
"username" seems confusing, this would be a component of the repository path regardless of its meaning to the server (e.g. org, user, etc).

Reply:
Actually, I used "username" because the commit that added the regression (https://github.com/go-git/go-git/commit/db0e226d48285a0ae8cda7db05d2ca20d9000dc6) used "username". I thought there was a good reason.

I will remove that "username" and use path, which is these cases' semantics (and link to upstream docs too).